packages/gatsby-cli/src/index.ts

```diff
@@ -17,17 +17,28 @@ if (useJsonLogger) {
 // Check if update is available
 updateNotifier({ pkg }).notify({ isGlobal: true })
 
-const MIN_NODE_VERSION = `>=8.0.0`
+const MIN_NODE_VERSION = `8.0.0`
+const NEXT_MIN_NODE_VERSION = `10.13.0`
 
-if (!semver.satisfies(process.version, MIN_NODE_VERSION)) {
+if (!semver.satisfies(process.version, `>=${MIN_NODE_VERSION}`)) {
   report.panic(
     report.stripIndent(`
-      Gatsby requires Node.js v8 or higher (you have ${process.version}).
+      Gatsby requires Node.js ${MIN_NODE_VERSION} or higher (you have ${process.version}).
       Upgrade Node to the latest stable release: https://gatsby.dev/upgrading-node-js
     `)
   )
 }
 
+if (!semver.satisfies(process.version, `>=${NEXT_MIN_NODE_VERSION}`)) {
+  report.warn(
+    report.stripIndent(`
+      Gatsby will be dropping support for Node.js ${process.version} soon 
```

Comment:
We should give a reason here -- that node 8 support in general is getting dropped as it's N years old

Reply:
The linked documentation page explains it in detail but fair enough. I'll update this to reflect that it's because they are no longer in maintained status.